$OpenBSD: patch-base_base_gyp,v 1.20 2013/04/16 14:55:20 ajacoutot Exp $

XXX clang has an internal fdatasync declaration which does not match

--- base/base.gyp.orig.port	Fri Mar 22 02:13:43 2013
+++ base/base.gyp	Tue Apr  2 21:05:17 2013
@@ -900,6 +900,13 @@
           ],
         }],
       ],
+      'conditions': [
+        [ 'OS == "openbsd"', {
+            # fdatasync is not implemented on OpenBSD
+            'sources/': [ ['exclude', '^test/test_file_util_linux.cc$'] ],
+          },
+        ],
+      ],
     },
   ],
   'conditions': [
